config/initializers/sidekiq.rb

```diff
@@ -1,7 +1,5 @@
 require 'sidekiq'
 
-Sidekiq::Extensions.enable_delay!
```

Comment:
Your commit message isn't consistent with what you changed here. You said `Sidekiq now disables delay extensions by default`, but this code was enabling the delay, not disabling it

Reply:
That's true, I miss interpreted the deprecation notice. It actually says that the Sidekiq Delayed Extensions will be removed on Sidekiq 7.0, whoever completely replacing it will probably need another task.

Reply:
No worries. But you should remove the commit [e201c4b](https://github.com/Codeminer42/cm42-central/pull/800/commits/e201c4b8957d4900bdc1169eba0ba1b17d3686eb) instead of creating a new one adding that code back.